timer/api/BUILD.bazel

```diff
@@ -0,0 +1,31 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
according to https://github.com/pingcap/tidb/issues/41241, we should put new package into ```pkg```.

Reply:
I have discussed with @bb7133 that this issue is still in progress and we can move all packages to `pkg` later when it is created.